terraform/environments/hmpps-domain-services/locals_test.tf

```diff
@@ -188,6 +188,180 @@ locals {
         }
       }
     }
+
+    baseline_lbs = {
+      networklb = {
+        internal_lb              = true
+        enable_delete_protection = false
+        load_balancer_type       = "application"
+        force_destroy_bucket     = true
+        subnets = [
+          module.environment.subnet["private"]["eu-west-2a"].id,
+          module.environment.subnet["private"]["eu-west-2b"].id,
+        ]
+        security_groups                  = ["load-balancer"]
+        access_logs                      = false
+        enable_cross_zone_load_balancing = true
+
+        instance_target_groups = {
+          rds-gateway-80 = {
+            port     = 80
+            protocol = "TCP"
+            health_check = {
+              enabled             = true
+              interval            = 5
+              healthy_threshold   = 3
+              port                = 80
+              protocol            = "TCP"
+              timeout             = 4
+              unhealthy_threshold = 2
+            }
+            stickiness = {
+              enabled = true
+              type    = "source_ip"
+            }
+            #attachments = [
+            #  { ec2_instance_name = "rds-gateway" },
+            #]
+          }
+          rds-gateway-7770 = {
```

Comment:
I'd get rid of these other ones 7770,7771,7780,7781 etc.. you won't need as they are CSR specific